### PR TITLE
fix(storefront): BCTHEME-708 fix wishlist dropdown shift in quickview modal

### PR DESCRIPTION
**Pull request by bc-alexsaiannyi**
#### What?
The issue relates to foundation dropdown lib miscalculation. Due to several  Wishlist’s dropdown on PDP once User makes attempt to open up one of them foundation.dropdown lib captures several ‘instances’ of them and makes calculation of css styles based on wrong ‘instance’. This leads that in quickview modal dropdown menu is separated  from button. As a solution we’ve added listener on foundation dropdown open Event  that rewrites  specific css style on its ‘original’ value. Once User leaves/closes  a Modal we clear our dropdown listener.

#### Requirements

- [x] CHANGELOG.md entry added (required for code changes only)

#### Tickets / Documentation

- [BCTHEME-708](https://jira.bigcommerce.com/browse/BCTHEME-708)

#### Screenshots (if appropriate)

https://user-images.githubusercontent.com/67792608/127316138-8fca4399-e53b-4589-ad2a-c7d351eb7289.mov

### CODE REVIEW

File: assets/js/theme/global/quick-view.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// import Foundation from 'foundation-sites';
```
Comment (BC-tymurbiedukhin) [inline assets/js/theme/global/quick-view.js:1]: Comment line

File: assets/js/theme/global/quick-view.js [hunk 2]
```diff
@@ -14,6 +15,14 @@ export default function (context) {
         event.preventDefault();
 
         const productId = $(event.currentTarget).data('productId');
+        const handleDropdown = ({ currentTarget }) => {
```
Comment (BC-tymurbiedukhin) [inline assets/js/theme/global/quick-view.js:18]: handleDropdown(Open | Expand)?

**Review by BC-tymurbiedukhin — APPROVED** [review]
LGTM

File: assets/js/theme/global/quick-view.js (hunk 1 above)
Comment (bc-alexsaiannyi) [reply assets/js/theme/global/quick-view.js]: my bad,  removed

File: assets/js/theme/global/quick-view.js (hunk 2 above)
Comment (bc-alexsaiannyi) [reply assets/js/theme/global/quick-view.js]: changed to handleDropdownExpand

**Review by bc-azvierieva — APPROVED** [review]

**Pull request merged**